include/battle_z_move.h

```diff
@@ -26,5 +26,6 @@ const u8 *GetZMoveName(u16 move);
 void SetZEffect(void);
 bool32 IsZMoveUsable(u8 battlerId, u16 moveIndex);
 void GetUsableZMoves(u8 battlerId, u16 *moves);
+u16 GetZMovePower(u16 move);
 
 #endif // GUARD_BATTLE_Z_MOVE_H
```

Comment:
You should add an empty line at the end of the file while you're at it :eyes:

Reply:
Done